generators/run/templates/run

```diff
@@ -319,14 +319,18 @@ update_dependencies() {
         fi
     fi
 
-    # Install pip dependecies
+    # Install python dependencies
     if [ -f requirements.txt ]; then
```

Comment:
Updating `./run` is only ever something we do manually in projects. At that point, it is trivial to run `pipenv install` to generate the `Pipfile{.lock}` and this is what I want us to do. Otherwise we'll never complete the transition.

Reply:
I'm not disagreeing with wanting it to be opinionated, just concerned someone will update ./run on a site which breaks a deployment that requires requirements.txt and then be very confused why.

Maybe this situation is just limited to Canonical and Ubuntu websites and its impact is small enough to be OK with it.

Reply:
No-one has ever updated the run script without submitting a PR and having it approved. That PR will fail in an extremely obvious way.

Reply:
Alright, we can go ahead with this. We do indeed want to block on the file, I was just concerned about the transition.